addon/helpers/list.js

```diff
@@ -2,6 +2,16 @@ import { MicroState } from 'ember-microstates';
 
 export default MicroState.extend({
 
+  initialValueFor([array]) {
+    if (array === undefined) {
+      return [];
+    } else if (array && array.length != null && array.forEach) {
```

Comment:
hmm... the problem is that `array.length != null` also catches strings. It'd have to be `!(array instanceof String) && array.legth != null`. Is that ok?